doc/source/operations/secret-rotation.rst

```diff
@@ -296,13 +304,17 @@ Full method
        ``00000000-0000-0000-0000-000000000000``, change the above command
        accordingly)
 
-
 15.  Re-encrypt your ``passwords.yml`` file
 
+16. Stop all OpenStack services
+
+    .. code:: bash
+
+       kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/stop-openstack-services.yml
 
 .. _k-a-change:
 
-16. Delete the service users in Keystone. The exact users will depend on the
+17. Delete the service users in Keystone. The exact users will depend on the
```

Comment:
hmm, actually we just stopped keystone...

Reply:
Maybe the best option is just to cherry-pick that KA patch. It makes it much smoother and we're already asking the user to make a change for the nova extended start

Reply:
Done